src/en/charms-storage.md

```diff
@@ -304,6 +305,15 @@ config:
   security.privileged: "true"
 ```
 
+### Remove storage
+
+Specify one or more unit storage, volume, or filesystem IDs, which can be found
+in the output of `juju storage`, like this:
+
+```bash
+juju remove-storage pgdata/0
+```
+
```

Comment:
- If the storage is attached to a charm, then we'll:
  - run the storage-detaching hook
  - detach the storage from the machine
  - remove the storage from the model (destroying the cloud storage resources in the process)
- If you remove a unit that has storage, nothing happens *yet*. Per [the spec](https://docs.google.com/document/d/1rl5NP8qWDD-wYgqmV-MlKPfvyL0I6BzbMZKf7n-tspw/edit#heading=h.fw0oe8lv4893), what we intend is for the storage to be detached (storage-detaching hook fires, and storage detaches from machine), but remain in the model. You'll then be able to redeploy the application with that storage, or remove the storage with this command.